src/app/designpad/implements.ts

```diff
@@ -28,3 +29,7 @@ export function implementsDraggable(object: any): object is Draggable {
 export function implementsDoubleClickable(object: any): object is DoubleClickable {
   return 'doubleClickEvent' in object;
 }
+
+export function implementsSelectable(object: any): object is SelectAble {
+  return 'selectEvent' in object;
+}
```

Comment:
Just so i understand. Is this used, so you can check if a given object have the SelectAble interface?

Reply:
Yes, that is it